### PR TITLE
re: Fix tag pattern for index command

### PR DESCRIPTION
**Pull request by mumoshu**
This is a successor or #23 that aims to follow up #16 and #17

### CODE REVIEW

File: pkg/index/main.go [hunk 1]
```diff
@@ -84,10 +84,10 @@ func Create(config *config.Options) error {
 		//fmt.Printf("found release %s\n", *r.TagName)
 		var packageName, packageVersion, packageURL string
 		for _, f := range r.Assets {
-			tagParts := strings.Split(*r.TagName, "+")
-			if len(tagParts) == 2 && *f.Name == fmt.Sprintf("%s-%s.tgz", tagParts[1], tagParts[0]) {
+			tagParts := splitPackageNameAndVersion(*r.TagName)
```
Comment (mumoshu) [inline pkg/index/main.go]: See https://github.com/helm/chart-releaser/pull/23#discussion_r258969572

File: pkg/index/main.go [hunk 2]
```diff
@@ -84,10 +84,10 @@ func Create(config *config.Options) error {
...
+			if len(tagParts) == 2 && *f.Name == fmt.Sprintf("%s-%s.tgz", tagParts[0], tagParts[1]) {
 				p := strings.TrimSuffix(*f.Name, filepath.Ext(*f.Name))
-				ps := strings.Split(p, "-")
+				ps := splitPackageNameAndVersion(p)
```
Comment (mumoshu) [inline pkg/index/main.go]: Without this, `chart-releaser` shows a slightly wrong message on `index`:

```
====> Using existing index at docs/index.yaml
--> Checking for releases with helm chart packages
====> Found kube-node.tgz
====> Found common-0.0.4.tgz
```

Here you see `Found kube-node.tgz`, but the actual asset file on the GitHub release is `kube-node-0.1.0.tgz`.

**Comment by unguiculus** [comment]
Thanks 🎉

**Comment by scottrigby** [comment]
Hey @mumoshu & @unguiculus I was on vacation, thanks for following up - I agree this fixes an issue that could be caused by #23

**Pull request merged**